Makefile

```diff
@@ -4,14 +4,14 @@ qa:
 	composer update --prefer-stable
 	PHP_CS_FIXER_IGNORE_ENV=true vendor/bin/php-cs-fixer fix --diff --verbose
 	vendor/bin/rector
-	vendor/bin/phpstan
+	vendor/bin/phpstan --memory-limit=-1
 	vendor/bin/phpunit
 
 
 qa-lowest:
 	composer update --prefer-lowest
 	PHP_CS_FIXER_IGNORE_ENV=true vendor/bin/php-cs-fixer fix --diff --verbose
-	vendor/bin/phpstan
+	vendor/bin/phpstan --memory-limit=-1
```

Comment:
Not sure why, but without this, I get:
```bash
     Child process error: PHPStan process crashed because it reached configured PHP memory limit: 128M
     Increase your memory limit in php.ini or run PHPStan with --memory-limit CLI option.
      while running parallel worker
     Child process error (exit code 255): PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in phar:///Users/oskar.stark/dev/php-llm/llm-chain/vendor/phpstan/phpstan/phpstan.phar/src/Analyser/MutatingScope.php on line 388
     Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in phar:///Users/oskar.stark/dev/php-llm/llm-chain/vendor/phpstan/phpstan/phpstan.phar/src/Analyser/MutatingScope.php on line 388
     PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in Unknown on line 0
     Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 262144 bytes) in Unknown on line 0
      while running parallel worker
```